src/components/images_uploading/DisplayImageList.js

```diff
@@ -1,36 +1,63 @@
-import React, { Component } from "react";
+import React, { Component, useReducer } from "react";
 import { connect } from "react-redux";
-import { fetchImages } from "./../../actions";
-import { Button } from "react-bootstrap";
+import { fetchImages, fetchUsers } from "./../../actions";
+import LocalApi from "../../apis/LocalApi";
+
+
+class UserAssignmentDropDown extends React.Component {

```

Comment:
it's temporary here, need to move to another file. This component shows a dropdown with list of users next to an image. When a user is selected in the dropdown an Assign button is shown.